sn_client/src/api/register_apis.rs

```diff
@@ -246,20 +246,16 @@ impl Client {
 }
 
 // temp dummy
-fn section_auth() -> sn_interface::messaging::SectionAuth {
-    use sn_interface::messaging::system::KeyedSig;
+fn section_auth() -> sn_interface::messaging::SectionSig {
```

Comment:
Rename the function too?
```suggestion
fn section_sig() -> sn_interface::messaging::SectionSig {
```